fedireads/goodreads_import.py

```diff
@@ -36,7 +35,7 @@ def construct_search_term(title, author):
     return ' '.join([title, author])
 
 
-class GoodreadsCsv(object):
```

Comment:
is this a more modern syntax? or is there a functional difference?

Reply:
There's no functional difference in Python 3, which is why pylint suggests the change. There is in Python 2, which is why I have a strong habit of always writing (object).

I think pylint is right as we never need to support python 2.